src/transformers/generation_beam_search.py

```diff
@@ -83,6 +85,41 @@
 """
 
 
+CONSTRAINT_PROCESS_INPUTS_DOCSTRING = r"""
```

Comment:
Same as before, this is only used once, so it should directly go on the method it documents.